src/main/java/seedu/foodrem/model/tag/UniqueTagList.java

```diff
@@ -17,7 +17,7 @@
  * A tag is considered unique by comparing using {@code Tag#equals(Tag)}. As such, adding and updating of
  * tags uses Tag#equals(Tag) for equality so as to ensure that the tag being added or updated is
  * unique in terms of identity in the UniqueTagList.
- *
+ * <p>
```

Comment:
Hmm, is this "<p>" tag necessary? I'm not sure if this is used anywhere else in FoodREM

Reply:
NOTE: Adds line break
![image](https://user-images.githubusercontent.com/53888954/195250041-f8d94477-6dd0-4349-a138-14fbe51338cc.png)